cardano-tx-generator/test/Main.hs

```diff
@@ -58,12 +58,12 @@ cliArgs = testGroup "cli arguments"
                         $ getParseResult $ execParserPure defaultPrefs (info commandParser fullDesc)
                            $ words l
 
-pinnedHelpMessage = [here|ParserFailure(Usage: <program> --config FILEPATH --socket-path FILEPATH 
```

Comment:
This is an exception because the trailing spaces belong to a string constant and are actually needed